tst/testinstall/mapping.tst

```diff
@@ -287,6 +287,14 @@ gap> IsGroupHomomorphism(res);
 true
 gap> IsInjective(res);        
 true
+gap> V := AsVectorSpace (GF(2), GF(2^17));;
+gap> h := BlownUpMat(Basis(V), [[Z(2^17)]]);;
+gap> ConvertToMatrixRep(h);;
+gap> G := CyclicGroup (2^17-1);;
+gap> H := Group (h);;
+gap> hom := GroupHomomorphismByImagesNC (G, H, [G.1], [h]);;
+gap> ImagesRepresentative(hom, G.1^2) = h^2;
+true
 gap> STOP_TEST( "mapping.tst", 5520000);
```

Comment:
I think it would be a good idea to add a comment before that test that explains the rationale. E.g.
```
gap> IsInjective(res);        
true

# The following test verifies we handle pcgs with large primes in them efficiently.
# See also GitHub pullrequest #576
gap> V := ...
```